gammapy/stats/variability.py

```diff
@@ -286,6 +286,78 @@ def structure_function(flux, flux_err, time, tdelta_precision=5):
     return sf, distances
 
 
+def discrete_correlation_function(
+    flux1, flux_err1, flux2, flux_err2, time1, time2, tau, axis=0
```

Comment:
Given the ordering of your docstring I would say it should be:

```suggestion
    flux1, flux2, flux_err1, flux_err2, time1, time2, tau, axis=0
```

Reply:
Whoops! I think it's better to change the docstrings in this case.